### ISSUE TITLE
Vagrant Up on Windows 7 not working

### ISSUE THREAD

**Issue by hydrosquall** [Swader/homestead_improved#6]
Following along with your tutorial [here](http://www.sitepoint.com/quick-tip-get-homestead-vagrant-vm-running/). Everything seems to work until the last chunk. Any idea how to debug it? Running Windows 7 with the latest versions of vagrant/virtualbox (just installed yesterday). 

```
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

chown `id -u vagrant`:`id -g vagrant` /vagrant

Stdout from the command:

Stderr from the command:

stdin: is not a tty
chown: changing ownership of ���/vagrant���: Not a directory
```

The preceding output before the above error looks fine:
```
C:\VM\vagrant_boxes\Homestead [master]> vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'laravel/homestead'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'laravel/homestead' is up to date...
==> default: Setting the name of the VM: Homestead_default_1437202627680_6988
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 80 => 8000 (adapter 1)
    default: 5000 => 5000 (adapter 1)
    default: 3306 => 33060 (adapter 1)
    default: 5432 => 54320 (adapter 1)
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default:
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default:
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default:
    default: Guest Additions Version: 4.3.14
    default: VirtualBox Version: 5.0
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /vagrant => C:/VM/vagrant_boxes/Homestead
```

My Homestead.yaml file looks like:
```yaml
---
ip: "192.168.10.10"
memory: 2048
cpus: 1

folders:
    - map: C:/Users/Cameron/Documents/GitHub/sass_step
      to: /home/vagrant/Code

sites:
    - map: myapp.com
      to: /home/vagrant/Code

variables:
    - key: APP_ENV
      value: local

```

Going to myapp.com doesn't return the file index.php inside the the sass_step directory.

**Comment by hydrosquall** [comment]
Tried a few other virtual boxes and found the issue was with Vagrant 1.7.3. The issue is supposed to be fixed when 1.7.4 rolls out, but until then, the fix on the Vagrant thread appears to work just fine.

Fix: https://github.com/mitchellh/vagrant/issues/5933#issuecomment-122508630

**Comment by Swader** [comment]
Love it when issues autoresolve :P
